### PR TITLE
PIL-1284 - added knowledge base in menu

### PR DESCRIPTION
**Pull request by vignesha22**

### CODE REVIEW

File: src/screens/Menu/Menu/components/SocialMediaLinks.js [hunk 1]
```diff
@@ -62,6 +62,11 @@ const SocialMediaLinks = () => {
   );
 };
 
+export const OpenKnowledgebase = () => {
+  const knowledgebaseUrl = firebaseRemoteConfig.getString(REMOTE_CONFIG.KNOWLEDGE_BASE_URL);
```
Comment (IAmKio) [inline src/screens/Menu/Menu/components/SocialMediaLinks.js:66]:
```suggestion
  const knowledgebaseUrl = firebaseRemoteConfig.getString(REMOTE_CONFIG.KNOWLEDGEBASE_URL);
```

**Review by IAmKio — CHANGES_REQUESTED** [review]
Just a few changes re: consistency

File: src/constants/remoteConfigConstants.js [hunk 2]
```diff
@@ -37,7 +37,7 @@ export const REMOTE_CONFIG = {
   FEATURE_ONBOARDING_TUTORIAL: 'feature_onboarding_tutorial',
   WALLET_MIGRATION_ARCHANOVA_ENABLED: 'feature_archanova_migrator_tool',
   ENS_MIGRATOR_ENABLED: 'ens_migrator_enabled',
-  KNOWLEDGE_BASE_URL: 'knowlegebase_url',
+  KNOWLEDGE_BASE_URL: 'knowledgebase_url',
```
[the remark this answers is not in the document]
Comment (vignesha22) [reply src/constants/remoteConfigConstants.js:40]: okay

**Review by IAmKio — APPROVED** [review]
LGTM - thanks for the changes

**Pull request merged**